tools/git-version.gradle

```diff
@@ -0,0 +1,24 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.ajoberstar:grgit:1.9.2'
+    }
+}
+
+import org.ajoberstar.grgit.Grgit
+
+ext {
+    git = Grgit.open(currentDir: projectDir)
+    gitVersionName = ""
+    gitVersionName = git.describe()
```

Comment:
Ummm we're using the tag naming "v1.2.3". This commands returns it all, so using it as artifact version would make it `com.schibsted.spain:barista:v1.2.3` which is not what we want.
We either remove the prefix from `git describe`, or change the tag naming for the project.